README.md

```diff
@@ -144,6 +144,28 @@ class FilteredSimplestModel(MasterMixin, models.Model):
         return len(str(self.name)) > 2
 ```
 
+Django Admin
+-----------
+
+Add action to syncronize master items from django admin page.
+
+```python
+from django.db import models
+from django.contrib import admin
+
+from dj_cqrs.admin_mixins import CQRSAdminMasterSyncMixin
+
+
+class AccountAdmin(CQRSAdminMasterSyncMixin, admin.ModelAdmin):
+    ...
+
+
+admin.site.register(models.Account, AccountAdmin)
+
+```
+
+* In case needed, we can override ```_cqrs_sync_queryset``` from CQRSAdminMasterSyncMixin to adjust the queryset to be used on syncing.
```

Comment:
If necessary, we can override ```_cqrs_sync_queryset``` from CQRSAdminMasterSyncMixin to adjust the QuerySet and use it for synchronization.